scripts/joinmarket-qt.py

```diff
@@ -1632,9 +1632,10 @@ def closeEvent(self, event):
     def initUI(self):
         self.statusBar().showMessage("Ready")
         self.setGeometry(300, 300, 250, 150)
-        loadAction = QAction('&Load...', self)
-        loadAction.setStatusTip('Load wallet from file')
-        loadAction.triggered.connect(self.selectWallet)
+        openWalletAction = QAction('&Open...', self)
+        openWalletAction.setStatusTip('Open joinmarket wallet file')
```

Comment:
nit - it's proper noun, so should be either "Joinmarket" or "JoinMarket" (my preference).
```suggestion
        openWalletAction.setStatusTip('Open JoinMarket wallet file')
```